Dropdown/Dropdown.js

```diff
@@ -184,37 +196,77 @@ const DropdownBase = kind({
 		}
 	},
 
-	render: ({children, css, dropdownListClassname, disabled, hasChildren, onClose, onOpen, onSelect, open, selected, transitionContainerClassname, transitionDirection, title, ...rest}) => {
+	render: ({children, css, dropdownButtonClassname, dropdownListClassname, disabled, hasChildren, onClose, onOpen, onSelect, open, selected, skin, transitionContainerClassname, transitionDirection, title, ...rest}) => {
 		const opened = !disabled && open;
+		const dropdownButton = (skin === 'silicon');
 
 		return (
 			<div {...rest}>
-				<Item
-					{...rest}
-					css={css}
-					disabled={hasChildren ? disabled : true}
-					onClick={opened ? onClose : onOpen}
-				>
-					<Icon slot="slotAfter" className={css.icon} size="small">{open ? 'arrowlargeup' : 'arrowlargedown'}</Icon>
-					{title}
-				</Item>
-				<Transition
-					className={transitionContainerClassname}
-					visible={opened}
-					direction={transitionDirection}
-				>
-					<ContainerDiv className={dropdownListClassname} spotlightDisabled={!open} spotlightRestrict="self-only">
-						<Scroller className={css.scroller}>
-							<Group
-								className={css.group}
-								childComponent={Item}
-								itemProps={{size: 'small'}}
-								onSelect={onSelect}
-								selected={selected}
-							>{children || []}</Group>
-						</Scroller>
-					</ContainerDiv>
-				</Transition>
+				{dropdownButton &&
+				<div className={dropdownButtonClassname}>
+					<Button
+						{...rest}
+						css={css}
+						disabled={hasChildren ? disabled : true}
+						onClick={opened ? onClose : onOpen}
+					>
+						{title}
+						<Icon slot="slotAfter" className={css.icon} size="small">{open ? 'arrowlargeup' : 'arrowlargedown'}</Icon>
+					</Button>
+					<Transition
+						className={transitionContainerClassname}
+						visible={opened}
+						direction={transitionDirection}
+					>
+						<ContainerDiv className={dropdownListClassname} spotlightDisabled={!open} spotlightRestrict="self-only">
+							<Scroller className={css.scroller}>
+								<Group
+									className={css.group}
+									childComponent={RadioItem}
+									itemProps={{size: 'small', className: css.dropDownListItem, css: css}}
+									onSelect={onSelect}
+									selected={selected}
+									selectedProp="selected"
+								>
+									{children || []}
+								</Group>
+							</Scroller>
+						</ContainerDiv>
+					</Transition>
+				</div>
+				}
+				{!dropdownButton &&
+					<React.Fragment>
```

Comment:
Do we need React.Fragment here?

Reply:
yes, we need an enclosing tag, But I did not want to add an unnecessary<div>